README.md

```diff
@@ -23,7 +23,7 @@
 
 ## About
 
-LibreLingo's mission is to create a modern language-learning platform that is owned by the community of its users. All software is licensed under AGPLv3, which guarantees the freedom to run, study, share, and modify the software. Course authors are encouraged to release their courses with free licenses.
```

Comment:
that wasn't a typo though.

also, how did you fix these typos? Did you run some automated check? I would appreciate if you could add this spelling check as part of the CI pipeline

Reply:
Hi, sorry. Yes, that one was was correct in the first place. Sorry about that. My bad. I've added another commit to resolve this.

No, I only read the text with my own two eyes and fixed typos where spotted. No automated checks =)